cmd/influx/remote.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
+	"github.com/urfave/cli"
+)
+
+func newRemoteCmd() cli.Command {
+	return cli.Command{
+		Name:   "remote",
+		Usage:  "Remote connection management commands",
+		Hidden: true, // Remove this line when all subcommands are completed
```

Comment:
Can you make an issue for this within the epic so we don't forget to do it before releasing?

Reply:
Yep, I'll do that now!